examples/Log/Log.ino

```diff
@@ -68,3 +69,10 @@ void loop() {
     Log.verbose  (F("Log as Verbose with bool value   : %T" CR CR CR               ), boolValue2);
     delay(5000);
 }
+
+void logPrefix(Print* _logOutput) {
```

Comment:
Can we name this `timePrefix` or similar? I think that makes it easier to understand that it is just a hook function.
And maybe we can add a suffix example with the newline by default?

Reply:
Good idea. I'll fix that and update the PR.

Reply:
Done